R/deployApp.R

```diff
@@ -227,6 +231,15 @@ deployApp <- function(appDir = getwd(),
     check_file(recordDir)
   }
 
+  if (!is_string(quarto)) {
```

Comment:
Is this check negated? Warn on an incoming string.

Reply:
Doh.